tools/_setup_generation/step_viselements.py

```diff
@@ -450,25 +450,25 @@ def setup(self, setup: Setup) -> None:
         self.generate_pages("blocks", self.blocks_path)
         self.generate_builder_api()
 
-    # Special case for charts: we want to insert the chart gallery that is stored in the
+    # Special case for charts: we want to insert the chart demos that is stored in the
     # file whose path is in self.charts_home_html_path
     # This should be inserted before the first level 1 header
     def chart_page_hook(
         self, element_documentation: str, before: str, after: str, charts_md_dir: str
     ) -> tuple[str, str]:
         with open(self.charts_home_html_path, "r") as html_fragment_file:
-            chart_gallery = html_fragment_file.read()
-            # The chart_gallery begins with a comment where all sub-sections
+            chart_demos = html_fragment_file.read()
```

Comment:
```suggestion
            chart_gallery = html_fragment_file.read()
```
and all the changes below